fluent-plugin-buffer-lightening.gemspec

```diff
@@ -17,5 +17,6 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.3"
   spec.add_development_dependency "rake"
+  spec.add_development_dependency "test-unit", "~> 3.1.7"
```

Comment:
`~> 3.0` looks better than it. I want not to depend minor version of such major library.